### PR TITLE
Added search bar and user can now search for the talks by name

### LINKED ISSUE #72: Add User to the Post table

### LINKED ISSUE #78: Search bar of Explore
This issue is related to user story #13.
Development of a search bar in order to fetch specific talks.

### PR DESCRIPTION
**Pull request by TitanicThompson1**
Closes #78

### CODE REVIEW

**Review by TiagooGomess — CHANGES_REQUESTED** [review]
You forgot to change the title :joy:

**Review by matosjoaops — CHANGES_REQUESTED** [review]
You're 71 commits behind master

**Comment by Jumaruba** [comment]
Must set the assignment, the reviewers, and the project in the bar on the right.

File: src/lib/main.dart [hunk 1]
```diff
@@ -11,13 +11,28 @@ Future<User> getCurrentUser() async => await FirebaseAuth.instance.currentUser;
...
   @override
   Widget build(BuildContext context) {
-    try {
-      if (FirebaseAuth.instance.currentUser != null) {
-        return MaterialApp(home: MyHome());
-      }
-      return MaterialApp(home: LoginScreen());
-    } catch (e) {
-      return MaterialApp(home: LoginScreen());
-    }
```
Comment (matosjoaops) [inline src/lib/main.dart:21]: Why is this being removed?

File: src/lib/main.dart [hunk 2]
```diff
@@ -11,13 +11,28 @@ Future<User> getCurrentUser() async => await FirebaseAuth.instance.currentUser;
...
+        primarySwatch: Colors.blue,
+        // This makes the visual density adapt to the platform that you run
+        // the app on. For desktop platforms, the controls will be smaller and
+        // closer together (more dense) than on mobile platforms.
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+      ),
+      home: MyHomePage(title: 'Flutter Demo Home Page'),
+    );*/
+
+    return MaterialApp(home: LoginScreen());
```
Comment (matosjoaops) [inline src/lib/main.dart:36]: And why is this being added back?

File: README.md [hunk 3]
```diff
@@ -91,16 +98,52 @@ So, what are you waiting for? Sign up today!
 * **Normal Flow**:
     * **User**: After the talk, clicks on the link given by
     * **System**: Opens the search submenu and presents the last searches from the user.
-    * **User**: Types and sends the name of the talk
-    * **System**: Searches in the database for the talk, and returns the results.
+    * **User**: Types and sends the name of the conference
+    * **System**: Searches in the database for the conference, and returns the results.
```
Comment (TiagooGomess) [inline README.md:102]: This pull request changed the README to a previous version

**Review by TiagooGomess — CHANGES_REQUESTED** [review]
Change the README version

**Comment by matosjoaops** [comment]
Several files are being reverted to previous versions, I think you did something wrong with git @TitanicThompson1

**Pull request closed without merging**